Article/Article.js

```diff
@@ -112,3 +116,152 @@ const data = [
+//     this.isOpen = false;
+//     const articleTitle = article.querySelector('h2');
+//     articleTitle.textContent = title;
+//     const articleDate = article.querySelector('.date')
+//     articleDate.textContent = date;
+//     this.articleButton = article.querySelector('.expandButton');
+//     this.articleContent = article.querySelector('.article');
+//     this.articleContent.textContent = firstParagraph, secondParagraph, thirdParagraph;
+//     this.articleButton.addEventListener('click', () => {
+//       this.toggleArticle();
+//     });
+//   }
+
+//   toggleButton() {
+//     this.isOpen = !this.isOpen;
+//     if (!this.isOpen) {
+//       this.articleButton.textContent = "Open";
+//     } else {
+//       this.articleButton.textContent = "Close";
+//     }
+//   }
+  
+  
+  
+//   toggleArticle() {
+//     this.toggleButton();
+//     this.articleContent.classList.toggle('article-open');
+//   }
+
+// }
+
+// // console.log(document.getElementsByClassName('articles')[0]);
+
+// const createNewArticle = () => {
+  
+//   const articles = document.querySelector('.articles');
+
+//   // console.log(articles)
+
+//   const newArticleSection = document.createElement('div');
+//   newArticleSection.classList.add('article');
+
+//   const newArticle = document.createElement('div');
+//   newArticle.classList.add('article');
+//   newArticleSection.appendChild(newArticle);
+
+//   const h2Article = document.createElement('h2');
+//   newArticle.appendChild(h2Article);
+
+//   const articleButton = document.createElement('expandButton');
+//   articleButton.classList.add('expandButton');
+//   articleButton.textContent = "Open";
+//   newArticle.appendChild(articleButton);
+
+//   const articleContent = document.createElement('div');
+//   articleContent.classList.add('date');
+//   newArticleSection.appendChild(articleContent);
+
+//   articles.appendChild(newArticleSection);
+
+//   console.log(newArticle);
+//   return newArticleSection;
+
+
+// }
+
+
+
+// data.map((article) => {
+//   const newArticleObject = new ArticlePanel(
+//     createNewArticle(),
+//     article.title,
+//     article.date,
+//     article.firstParagraph,
+//   );
+// })
+
+
+
+
+
+
+
+function articleTemplate(articles) {
+  const article = document.createElement('div');
+  
+  const title = document.createElement('h2');
+  article.appendChild(title);
+  title.textContent = articles.title;
+  
+  const date = document.createElement('p');
+  article.appendChild(date);
+  date.textContent = articles.date
+ 
+  const firstPara = document.createElement('p');
+  article.appendChild(firstPara);
+  firstPara.textContent = articles.firstParagraph;
+ 
+  const secondPara = document.createElement('p');
+  article.appendChild(secondPara);
+  secondPara.textContent = articles.secondParagraph;
+ 
+  const thirdPara = document.createElement('p');
+  article.appendChild(thirdPara);
+  thirdPara.textContent = articles.thirdParagraph;
```

Comment:
This works, if you ever have free time and come back to this I'd look into making it DRY